Python/preconfig.c

```diff
@@ -758,6 +758,7 @@ _PyPreConfig_ReadFromArgv(_PyPreConfig *config, const _PyArgv *args)
 done:
     if (init_ctype_locale != NULL) {
         setlocale(LC_CTYPE, init_ctype_locale);
+        PyMem_RawFree(init_ctype_locale);
```

Comment:
Oops, I missed that in the big refactoring I did last ones. Nicely spotted ;-)

Reply:
There's a similar leak in _PyPreConfig_Read, https://github.com/btharper/cpython/commit/81e617d18b1f64f24895f5c83fadfc655f05d760 I didn't see any more cases grepping for setlocale calls